ipa-server/.applier/group_vars/seed-hosts.yml

```diff
@@ -0,0 +1,64 @@
+---
+namespace: ipa
+app_name: ipa
+base_openshift_url: example.com 
+realm: "EXAMPLE.COM"
+admin_password: "Password123"
+openshift_templates_version: "v1.4.5"
+
+
+params:
+  IPA_SERVER_SERVICE: "{{ app_name }}"
+  IPA_SERVER_IMAGE: "freeipa-server:rhel-7"
+  IPA_SERVER_HOSTNAME: "{{ app_name }}.apps.{{ base_openshift_url }}"
+  IPA_SERVER_INSTALL_OPTS: "-U -r {{ realm }} --setup-dns --no-forwarders --no-ntp"
+  IPA_ADMIN_PASSWORD: "{{ admin_password }}"
+  SERVICE_ACCOUNT_USEROOT: "{{ app_name }}-sa"
+  VOLUME_CAPACITY: 1G
+  TIMEOUT: 600
+
+openshift_cluster_content:
+- object: Environment Setup
+  content:
+  - name: Create Projects
+    template: "https://raw.githubusercontent.com/redhat-cop/openshift-templates/{{ openshift_templates_version }}/project-requests/create-project.yml"
+    action: create
+    params_from_vars:
+      NAMESPACE: "{{ namespace }}"
+      NAMESPACE_DISPLAY_NAME: "{{ namespace }}"
+    tags:
+      - ipa_project
+
+- object: Image Streams
+  content:
+  - name: "{{ app_name }}-streams"
+    file: "https://raw.githubusercontent.com/freeipa/freeipa-container/5e55ee8a62e7c51bf8c1805b33425f48e2d8f44e/freeipa-server-openshift-image.json"
```

Comment:
Does it make sense to use something like `{{ freeipa_templates_version }}` to prepare for future updates?

Reply:
Done